### ISSUE TITLE
Simplify ULP Definitions for Geometric Functions

### ISSUE THREAD

**Issue by bashbaug** [KhronosGroup/OpenCL-Docs#150]
Pull request #43 defined ULP definitions for geometric functions, and pull request #142 fixed a few asciidoc quirks in the way the ULP definitions needed to be formatted.

The ULP definitions are a little complicated though, for example:

Function | ULP
---------| ---
`distance` | 3 + (1.5 * n) + (0.5 * (n - 1)) ulp, for gentype with vector width n

Is there any reason this cannot be simplified?

```
3 + (1.5 * n) + (0.5 * (n - 1))
= 3 + 1.5n + (0.5n - 0.5)
= 2.5 + 2n
```

This example uses `distance`, but I think this could apply to many (if not all) of the geometric functions.

**Comment by EwanC** [comment]
Think presenting the final simplified ULP is defintely more readible for users.

As a side-effect I'm wondering if it's worth also record somewhere in this repo how we arrived at these expressions for our own reference. Currently the working is in CTS comments, which adds a level
of indirection which could go stale. As in the case of my [!43](https://github.com/KhronosGroup/OpenCL-Docs/pull/43) MR links which point to
the cl22_trunk CTS branch which no longer exists.

Could copy the CTS comments into asciidoc comments, e.g for `distance()`.
```c
 float maxUlps = 3.0f +                              // error in sqrt
        ( 1.5f * (float) sizes[size] +      // cumulative error for multiplications  (a-b+0.5ulp)**2 = (a-b)**2 + a*0.5ulp + b*0.5 ulp + 0.5 ulp for multiplication
         0.5f * (float) (sizes[size]-1));    // cumulative error for additions
```

```
////
3           ULP error in sqrt
1.5 * n     cumulative error for multiplications (a-b+0.5ulp)**2 = (a-b)**2 + a*0.5ulp + b*0.5 ulp + 0.5 ulp for multiplication
0.5 * (n-1) cumulative error for additions

= 3 + (1.5 * n) + (0.5 * (n - 1))
= 3 + 1.5n + (0.5n - 0.5)
= 2.5 + 2n
////
| *distance*            | {leq} 2.5 + 2n ulp, for gentype with vector width n

```

**Comment by bashbaug** [comment]
Fixed by #194 - closing.
